test/integration/framework/framework.go

```diff
@@ -110,6 +110,15 @@ func DeleteQueue(ctx context.Context, c client.Client, q *kueue.Queue) error {
 	return nil
 }
 
+func DeleteWorkload(ctx context.Context, c client.Client, wl *kueue.QueuedWorkload) error {
```

Comment:
We already delete all the QWs when we delete the namespace (see below)

Reply:
ok will remove this part